src/AutoRest.CSharp/MgmtTest/TestCommon/Extensions.cs

```diff
@@ -0,0 +1,176 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using AutoRest.CSharp.Input;
+using AutoRest.CSharp.Mgmt.Output;
+using AutoRest.CSharp.Mgmt.AutoRest;
+using AutoRest.CSharp.Mgmt.Models;
+using System.Reflection.Metadata.Ecma335;
+using AutoRest.CSharp.MgmtTest.Models;
+
+namespace AutoRest.CSharp.MgmtTest.TestCommon
+{
+    internal static class Extensions
```

Comment:
Since this class contains the extension methods, could we separate them by which type is it extending on? Like `StringExtensions`, `OavVraiableScopeExtensions`?